### PR TITLE
AAAAHH I FINISHED EVERYTHING BEFORE NOON DON'T THROW EGGS AT ME

### PR DESCRIPTION
**Pull request by k4therin2**

### CODE REVIEW

**Comment by thomasrussellmurphy** [comment]
GitHub has highlighted some `&` that need to be escaped in the HTML.

Edit: also, some files don't have trailing newlines :P

File: index.html [hunk 1]
```diff
@@ -87,53 +90,47 @@
...
+<hr>
+        <div class="row">
+          <div class="col-lg-4"> <img src="img/hackathon.jpg" class="img-responsive img-rounded" ></div>
+          <div class="col-lg-8">
+            <h2>Well, first of all, we require like, <strong>0 commitment</strong> from you. You don't "join" hacsoc on account of we think that that whole club model is ridiculous. We know that you have classes and other commitments. All you have to do is show up when you want to, for the talks or events you're interested in.  </h2></div>
+ </div>
+ <div class="row">
+ <div class="col-lg-12">
+
+                        <h2><strong>If the beauty of a noncomittal relationship</strong> isn't enough for you, I'll bring up the fact that a ton of <strong>our club members and alumni work and intern at companies like Google, Microsoft, Amazon and various local startups</strong>. For recruiters, seeing "hacker society" on a resume means that that canditate goes above and beyond the classroom to get involved in the field. We're not the students that close our laptop after we get out of class- we're students who are excited and passionate about the applications of what we're learning.</h2>
```
Comment (brenns10) [inline index.html]: `s/canditate/candidate/`

File: index.html [hunk 2]
```diff
@@ -87,53 +90,47 @@
...
+            <h2>The importance of getting involved in outside projects, hackathons, and other events is invaluable to your future career. Trust us. <strong>You'll learn stuff from chilling with us that you're not going to learn in class</strong> (your professors will generally be teaching theory, and we'll be showing you real-life application).</h2>
+
+            </div>     
+        </div>
+<h1>Gee whiz, sounds like a deal! Where do I sign, Mr. Berry Gordy?<h1>
+<hr>
+        <div class="row">
+          <div class="col-lg-4"> <img src="img/color.jpg" class="img-responsive img-rounded" ></div>
+          <div class="col-lg-8">
+          <h2>To get started, sign up for our <a href="https://lists.case.edu/wws/info/cwru-hackers?checked_cas=0&ticket=ST-5383-b7wjNUrejVJNcQ7gvB93-ssop-cfd-1.cwru.edu" style="font"><u>mailing list</u></a>. The way we operate is incredibly old school: <strong>we send out an newsletter/email thing at the beginning of the week</strong> with all the details of that week's events. The newsletter will also be posted here, but join the list anyway because that's where last-minute opportunities and job offers are sent out.</h2>
```
Comment (brenns10) [inline index.html]: This appears to be a CAS session token that you'll want to get rid of before deploying...

File: about.html [hunk 3]
```diff
@@ -41,10 +43,11 @@
                 <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                     <span class="sr-only">Toggle navigation</span>
                     <span class="icon-bar"></span>
- 				    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
```
Comment (ajm188) [inline about.html:44]: nitpick: is this github showing me a tab? spaces >>>>> tabs

File: about.html [hunk 4]
```diff
@@ -76,10 +79,10 @@
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                     <div class="site-heading">
-                    	<h3>Case Western Reserve</h3>
+                        <h3>Case Western Reserve</h3>
```
Comment (ajm188) [inline about.html:80]: nitpick: spaces >>>>> tabs

File: index.html (hunk 1 above)
Comment (k4therin2) [inline index.html]: ??

Comment (ajm188) [inline index.html]: substitution. it means "replace 'canditate' with 'candidate'"

**Comment by StephHippo** [comment]
@k4therin2 is this good to go or are you still making edits?

**Comment by k4therin2** [comment]
@brenns10 @thomasrussellmurphy @StephHippo @ajm188 fixed everything up hopefully we're good to go now

File: events.html [hunk 5]
```diff
@@ -138,4 +150,4 @@
 
 </body>
 
-</html>
+</html>
```
Comment (ajm188) [inline events.html]: no newline at end of file

File: css/clean-blog.min.css [hunk 6]
```diff
@@ -69,4 +74,10 @@ li a :hover {
...
 
-strong :hover {color: #00EE00 !important;}
\ No newline at end of file
+strong :hover {color: #00EE00 !important;}
+
+.navbar-brand>img {
+  display: inline;
+}
+
+h2 {font-family: 'Open Sans'; font-weight: normal;}
```
Comment (ajm188) [inline css/clean-blog.min.css:83]: no newline at end of file

**Comment by thomasrussellmurphy** [comment]
Good stuff. It'll probably be easier to dig into the updating once we have something live.

+1

**Pull request merged**